<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerifyCode</class>
 <widget class="QWidget" name="VerifyCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">width: 500px;
height: 629px;
line-height: 20px;
background-color: rgb(17, 23, 41);
font-size: 14px;
text-align: center;
box-shadow: 0px 12px 6px 0px rgba(0, 0, 0, 0.44);
font-family: Roboto;</string>
  </property>
  <widget class="QLabel" name="flag">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/img/flag.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 180px;
height: 44px;
font-size: 30px;
text-align: left;
font-family: SourceHanSansSC-regular;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>验证手机号码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SourceHanSansSC-regular</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 222px;
height: 26px;
font-size: 18px;
text-align: left;
font-family: SourceHanSansSC-regular;
color: rgb(255, 64, 64);</string>
   </property>
   <property name="text">
    <string>验证码输入有误，请重新输入</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 60px;
height: 60px;
line-height: 20px;
border-radius: 10px;
text-align: center;
background-color: rgb(70, 105, 232);
color: rgb(255, 255, 255);
font-size: 30px;
font-family: SourceHanSansSC-regular;</string>
      </property>
      <property name="maxLength">
       <number>1</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 60px;
height: 60px;
line-height: 20px;
border-radius: 10px;
text-align: center;
background-color: rgb(70, 105, 232);
color: rgb(255, 255, 255);
font-size: 30px;
font-family: SourceHanSansSC-regular;</string>
      </property>
      <property name="maxLength">
       <number>1</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
font-family: SourceHanSansSC-regular;
width: 60px;
height: 60px;
line-height: 20px;
border-radius: 10px;
text-align: center;
background-color: rgb(70, 105, 232);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="maxLength">
       <number>1</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
font-family: SourceHanSansSC-regular;
width: 60px;
height: 60px;
line-height: 20px;
border-radius: 10px;
text-align: center;
background-color: rgb(70, 105, 232);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="maxLength">
       <number>1</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
font-family: SourceHanSansSC-regular;
width: 60px;
height: 60px;
line-height: 20px;
border-radius: 10px;
text-align: center;
background-color: rgb(70, 105, 232);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="maxLength">
       <number>1</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
font-family: SourceHanSansSC-regular;
width: 60px;
height: 60px;
line-height: 20px;
border-radius: 10px;
text-align: center;
background-color: rgb(70, 105, 232);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="maxLength">
       <number>1</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 126px;
height: 26px;
font-size: 18px;
text-align: left;
font-family: SourceHanSansSC-regular;
color: rgb(79, 105, 223);
background-color: rgb(17, 23, 41);</string>
   </property>
   <property name="text">
    <string>获取验证码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 194px;
height: 60px;
line-height: 41px;
border-radius: 15px;
opacity:0.4;
font-size: 28px;
text-align: center;
font-family: Roboto;
background-color: rgb(79, 105, 223);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>上一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 194px;
height: 60px;
line-height: 41px;
border-radius: 15px;

font-size: 28px;
text-align: center;
font-family: Roboto;
background-color: rgb(79, 105, 223);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SourceHanSansSC-regular</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 414px;
height: 26px;

font-size: 18px;
text-align: left;
font-family: SourceHanSansSC-regular;
color: rgb(133, 135, 155);</string>
   </property>
   <property name="text">
    <string>点击下一步代表你已阅读并同意服务协议和隐私政策</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_send">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SourceHanSansSC-regular</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 414px;
height: 53px;
color: rgb(178, 180, 202);
font-size: 16px;
text-align: left;
font-family: SourceHanSansSC-regular;</string>
   </property>
   <property name="text">
    <string>请输入发送至188XXXX8888的6位验证码，有效期10分钟。</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
